tests/conftest.py

```diff
@@ -25,6 +25,7 @@
 import pytest
 import webtest as _webtest
 
+from psycopg.errors import InvalidCatalogName
```

Comment:
```suggestion
from psycopg2.errors import InvalidCatalogName
```
We currently have `psycopg2` as a dependency. Is it necessary to install `psycopg` as well?

Reply:
@di It is to upgrade to `pytest-postgresql` to 4.1.0. Since version 4 and upwards it supports only psycopg 3 and DatabaseJanitor's connections are separate from any other, and it does not interfere with psycopg2 installation.

On the other hand, it might confuse developers and accidentally import something from psycopg 3 and not 2...

Reply:
I see. I think I'd prefer to upgrade from pyscopg2 to 3 first, otherwise this would be somewhat confusing.

Reply:
Okay then... I'll limit the bump of pytest-postgresql to 3.1.2 then... Will also see how to make dependabot hush about the updates on pytest-postgresql for the time being 🤔

Reply:
@di done, along with the fixture fix